storyblok/storyblok-design-system
# refactor: sb tabs

## Pull request type

Jira Link: -




## How to test this PR


Compare:

- https://blok.ink/?path=/docs/design-system-components-sbtabs--default
- https://storyblok-design-system-git-refactor-ar-33-2e9acb-storyblok-com.vercel.app/?path=/docs/design-system-components-sbtabs--default


It's also good to check in the app:
```cd storyblok-design-system && git checkout refactor/ar-33-sb-tabs```
Then start the server of the app


## What is the new behavior?



-  it uses the template syntax now in the inner components. For the outer ones, we have to keep the rendering api for now
- it adds tab navigation and outline colors
- 

## Other information
